ansible/roles/tenderdash/templates/tenderdash/genesis.json.j2

```diff
@@ -0,0 +1,33 @@
+{
+  "genesis_time": "{{ genesis_time }}",
+  "chain_id": "dash-{{ ( 'devnet-' + dash_devnet_name if dash_network == 'devnet' else dash_network ) if tenderdash_chain_id is not defined else tenderdash_chain_id }}",
+  "consensus_params": {
+    "timeout": {
```

Comment:
Initial core chain lock height missed?

Reply:
Good catch, although it doesn't seem to be necessary?

Reply:
I'm not sure we need something except chain_id and genesis time for seed nodes at all

Reply:
`validator_quorum_type` is needed as well, everything else successfully removed